Source/WTF/wtf/PlatformHave.h

```diff
@@ -1285,6 +1285,10 @@
 #define HAVE_MEDIA_VOLUME_PER_ELEMENT 1
 #endif
 
+#if PLATFORM(IOS_FAMILY)
+#define HAVE_FULLSCREEN_DISMISSAL_GESTURES 1
```

Comment:
I think this needs to be moved to `PlatformEnableCocoa` with a `#if !defined(ENABLE_FULLSCREEN_DISMISSAL_GESTURES)`